smrf/tests/smrf_test_case.py

```diff
@@ -129,3 +130,7 @@ def tearDownClass(cls):
         folder = os.path.join(cls.base_config.cfg['output']['out_location'])
         if os.path.exists(folder):
             shutil.rmtree(folder)
+
+        logging.shutdown()
+        for handler in logging.root.handlers[:]:
+            logging.root.removeHandler(handler)
```

Comment:
Make this a class method so you can also call it from `SMRFTestCaseLakes` and remove duplicated code?

Reply:
I'm not entirely sure why this all of a sudden was required, we've never needed it before until now. I'm hoping to pull the logging part of `model_framework` into it's own class to clean up our logging in SMRF then this would be a good method to have there.